### ISSUE TITLE
speedups.c is missing from the 11.0 source distribution

### ISSUE THREAD

**Issue by bdraco** [aaugustin/websockets#1329]
Thanks for releasing 11.0.  It looks like there is a small issue with building wheels.

https://pypi.org/project/websockets/#files https://files.pythonhosted.org/packages/06/e7/de79d7cdde692dedb0944c42a03c358118f13cdff79d016a0504dbca35e4/websockets-11.0.tar.gz

https://github.com/aaugustin/websockets/actions/runs/4587678554/jobs/8101352087 seems to have missed it somehow

```
% tar ztvf websockets-11.0.tar.gz | grep speedup
%
```

Building wheels fails
```
    gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fno-semantic-interposition -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -fno-builtin-free -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/local/include/python3.10 -c src/websockets/speedups.c -o build/temp.linux-x86_64-cpython-310/src/websockets/speedups.o
      cc1: fatal error: src/websockets/speedups.c: No such file or directory
      compilation terminated.
      error: command '/usr/bin/gcc' failed with exit code 1
```

**Comment by aaugustin** [comment]
Indeed speedups.c was included in the 10.4 sdist and is missing from the 11.0 sdist.

It might be an unintended consequence of this:

https://github.com/aaugustin/websockets/blob/fff80c316e25af645ab5cc9b7217b2064d429d73/setup.py#L22-L23

and that:

https://github.com/aaugustin/websockets/blob/fff80c316e25af645ab5cc9b7217b2064d429d73/.github/workflows/wheels.yml#L13-L14

I will fix it and push a new version (11.0.1).

**Comment by aaugustin** [comment]
Fixed in f55c1411a780d2829f095d426707860b90ff3c0c. 11.0 was the only version affected by this issue.

**Comment by aaugustin** [comment]
Build logs showing that speedups.c is included in the 11.0.1 sdist: https://github.com/aaugustin/websockets/actions/runs/4625697877/jobs/8181697145#step:4:52

**Comment by bdraco** [comment]
Thanks. Wheel builds are working now https://wheels.home-assistant.io/musllinux/
